tests/app/main/views/test_users.py

```diff
@@ -283,3 +284,155 @@ def test_download_csv(self, data_api_client):
+                "createdAt": "Thu, 04 Aug 2016 12:00:00 GMT"
+            }
+        ]
+
+        data_api_client.find_briefs_iter.return_value = [
+            {
+                'title': 'This is a brief',
+                'status': 'draft',
+                'users': [{
+                    'id': 1
+                }]
+            },
+            {
+                'title': 'This is a second brief',
+                'status': 'draft',
+                'users': [{
+                    'id': 1
+                }]
+            }
+        ]
+
+        response = self.client.get('/admin/users/download/buyers')
+        rows = [line.split(",") for line in response.get_data(as_text=True).splitlines()]
+        buyer = rows[1]
+
+        assert response.status_code == 200
+        assert len(rows) == 2
+        assert buyer == [u'Chris', u'chris@gov.uk', u'01234567891', u'"Thu',
+                         u' 04 Aug 2016 12:00:00 GMT"',
+                         u'This is a brief - draft; This is a second brief - draft']
+
+    def test_buyer_is_listed_if_they_have_no_briefs(self, data_api_client):
+        data_api_client.find_users_iter.return_value = [
+            {
+                'id': 1,
+                "name": "Chris",
+                "emailAddress": "chris@gov.uk",
+                "phoneNumber": "01234567891",
+                "createdAt": "Thu, 04 Aug 2016 12:00:00 GMT"
+            }
+        ]
+
+        data_api_client.find_briefs_iter.return_value = [
+            {
+                'title': 'This is a brief',
+                'status': 'draft',
+                'users': [{
+                    'id': 2
+                }]
+            }
+        ]
+
+        response = self.client.get('/admin/users/download/buyers')
+        rows = [line.split(",") for line in response.get_data(as_text=True).splitlines()]
+        buyer = rows[1]
+
+        assert response.status_code == 200
+        assert buyer == [u'Chris', u'chris@gov.uk', u'01234567891', u'"Thu',
+                         u' 04 Aug 2016 12:00:00 GMT"', '']
+
+    def test_response_is_a_csv(self, data_api_client):
+        data_api_client.find_users_iter.return_value = [
+            {
+                'id': 1,
+                "name": "Chris",
+                "emailAddress": "chris@gov.uk",
+                "phoneNumber": "01234567891",
+                "createdAt": "Thu, 04 Aug 2016 12:00:00 GMT"
+            }
+        ]
+
+        data_api_client.find_briefs_iter.return_value = [
+            {
+                'title': 'This is a brief',
+                'status': 'draft',
+                'users': [{
+                    'id': 1
+                }]
+            }
+        ]
+
+        response = self.client.get('/admin/users/download/buyers')
+
+        assert response.mimetype == 'text/csv'
+
+    def test_filename_includes_a_timestamp(self, data_api_client):
+        with mock.patch('app.main.views.users.datetime') as mock_date:
```

Comment:
I think an alternative here would be to use `freeze_time`. You can see an example [here](https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/0578507db1d2cfb23b55f177f4d74f1de69bd3ce/tests/app/main/test_services.py#L798). I think your current solution is still good though but good to be aware of freeze_time.